js/stdlib/Makefile

```diff
@@ -26,24 +30,24 @@ build: package.json
 	  --build-arg NODE_VERSION=$(NODE_VERSION) \
 	  --build-arg REACH_VERSION=$(VERSION) \
 	  --build-arg REACH_GIT_HASH=$(REACH_GIT_HASH) \
-		.
+	  .
 	TAG_ONLY=1 $(ROOT)/scripts/docker-push.sh $(IMAGE)
 
-.PHONY: run
-run: build
-	docker-compose run test-stdlib-ETH
-	docker-compose run test-stdlib-ALGO
-	docker-compose run test-stdlib-FAKE
-	docker-compose run test-rpc-server-ETH
-	docker-compose run test-rpc-server-ALGO
-	docker-compose run test-rpc-server-FAKE
-
-.PHONY: down
-down:
-	docker-compose down
-
+# Note: We're creating an `stdlib-test` directory from scratch here in order
+#   to leverage the `reach` script and abstract over boilerplate such as
+#   `docker-compose.yml` config.
 .PHONY: test
-test: run
+test: build clean-test
+	mkdir test/stdlib-test
+	cp    test/index.* test/stdlib-test
+	cp -r test/lib     test/stdlib-test/lib
+	cd test/stdlib-test \
+	  && reach scaffold \
```

Comment:
Since you are not patching it, this is not necessary

Reply:
Thanks - fixed in a4d0681f7c934591e14556e44afe4c0a9578ded9.